### PR TITLE
Fix: AI - HTML thumbnail is not scaled and translated correctly [ED-12735]

### PR DESCRIPTION
**Pull request by matipojo**

### CODE REVIEW

File: modules/ai/assets/js/editor/pages/form-layout/components/attachments/thumbnail.js [hunk 1]
```diff
@@ -2,17 +2,25 @@ import { useEffect, useRef } from 'react';
...
 	useEffect( () => {
 		if ( previewRef.current ) {
 			const previewRoot = previewRef.current.firstElementChild;
+
 			const width = previewRoot?.offsetWidth || THUMBNAIL_SIZE;
 			const height = previewRoot?.offsetHeight || THUMBNAIL_SIZE;
 
+			// Keep the aspect ratio
+			previewRoot.style.width = `${ width }px`;
+			previewRoot.style.height = `${ height }px`;
```
Comment (StyleShit) [inline modules/ai/assets/js/editor/pages/form-layout/components/attachments/thumbnail.js:19]: not sure that I understand the issue, but maybe `object-fit: cover;`?

File: modules/ai/assets/js/editor/pages/form-layout/components/attachments/thumbnail-url.js
Comment (StyleShit) [inline modules/ai/assets/js/editor/pages/form-layout/components/attachments/thumbnail-url.js:1]: not related to this PR, but line 39 is missing translation domain

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: modules/ai/assets/js/editor/pages/form-layout/components/attachments/thumbnail.js (hunk 1 above)
Comment (matipojo) [reply modules/ai/assets/js/editor/pages/form-layout/components/attachments/thumbnail.js:19]: It's HTML scaled to the thumbnail size, so the `object-fit` will not work

File: modules/ai/assets/js/editor/pages/form-layout/components/attachments/thumbnail-url.js
Comment (matipojo) [reply modules/ai/assets/js/editor/pages/form-layout/components/attachments/thumbnail-url.js:1]: Fixed

**Review by StyleShit — APPROVED** [review]

**Pull request merged**